Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```diff
@@ -200,44 +216,233 @@ script:
+      required: false
+    - description: Number of results in a page. default is 50, the limit for API V1 is 1000, the limit for API V2 is 2000.
+      isArray: false
+      name: page_size
+      required: false
+    - description: Number of total results to return. default is 50.
+      isArray: false
+      name: limit
+      required: false
+      defaultValue: 50
     deprecated: false
     description: List alerts (security issues) within a customer's tenant that Microsoft or partner security solutions have identified.
     execution: false
     name: msg-search-alerts
     outputs:
     - contextPath: MsGraph.Alert.ID
-      description: Alert ID
+      description: Relevant only for V1. Alert ID
       type: string
     - contextPath: MsGraph.Alert.Title
-      description: Alert title
+      description: Relevant only for V1. Alert title
       type: string
     - contextPath: MsGraph.Alert.Category
-      description: Alert category
+      description: Relevant only for V1. Alert category
       type: string
     - contextPath: MsGraph.Alert.Severity
-      description: Alert severity
+      description: Relevant only for V1. Alert severity
       type: string
     - contextPath: MsGraph.Alert.CreatedDate
-      description: Alert created date
+      description: Relevant only for V1. Alert created date
       type: date
     - contextPath: MsGraph.Alert.EventDate
-      description: Alert event time
+      description: Relevant only for V1. Alert event time
       type: date
     - contextPath: MsGraph.Alert.Status
-      description: Alert status
+      description: Relevant only for V1. Alert status
       type: string
     - contextPath: MsGraph.Alert.Vendor
-      description: Alert vendor/provider
+      description: Relevant only for V1. Alert vendor/provider
       type: string
     - contextPath: MsGraph.Alert.MalwareStates
-      description: Alert malware states
+      description: Relevant only for V1. Alert malware states
       type: string
     - contextPath: MsGraph.Alert.Vendor
-      description: Alert vendor
+      description: Relevant only for V1. Alert vendor
       type: string
     - contextPath: MsGraph.Alert.Provider
-      description: Alert provider
+      description: Relevant only for V1. Alert provider
       type: string
+    - contextPath: MsGraph.Alert.actorDisplayName
+      description: Relevant only for V2. Alert actor name
+      type: Unknown
+    - contextPath: MsGraph.Alert.alertWebUrl
+      description: Relevant only for V2. Alert web url
+      type: String
+    - contextPath: MsGraph.Alert.assignedTo
+      description: Relevant only for V2. Alert assignee
+      type: Unknown
+    - contextPath: MsGraph.Alert.category
+      description: Relevant only for V2. Alert category
+      type: String
+    - contextPath: MsGraph.Alert.classification
+      description: Relevant only for V2. Alert classification
+      type: Unknown
+    - contextPath: MsGraph.Alert.createdDateTime
```

Comment:
whats the difference between contextPath: MsGraph.Alert.CreatedDate and this 1?

Reply:
The c in created.. it's basically the same information but @DanilVil said to return the context output as is for V2 unlike V1 where we take only certain fields.